.github/workflows/flake8.yml

```diff
@@ -8,12 +8,12 @@ jobs:
     name: flake8 Linter
     steps:
       - name: Check out source repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Set up Python environment
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v3
```

Comment:
just curious if there was a problem with these or if this is just the newer versions?

Reply:
Just newer versions. Figured I might as well update it while I was in there. The old ones work fine too.